examples/react-client/use-camera-and-microphone/README.md

```diff
@@ -4,19 +4,19 @@
 
 Follow these steps to start and run your project:
 
-1. Install Dependencies: Use npm to install the project dependencies:
+1. Install Dependencies: Use yarn to install the dependencies in root directory:
 
 ```shell
 npm install
 ```
```

Comment:
Use yarn to install and then command uses npm? 
It should be consistent with main readme, so one command for yarn and one for npm.

Reply:
I missed that line, fixed